.gitlab-ci.yml

```diff
@@ -178,3 +184,19 @@ deploy_to_debugger_backend:
     UPSTREAM_BRANCH: $CI_COMMIT_BRANCH
     UPSTREAM_TAG: $DEPLOY_TAG
 
+
+package:
+  extends: .package
+  rules:
+    - if: $DOTNET_PACKAGE_VERSION
+      when: on_success
+    - if: '$CI_COMMIT_TAG =~ /^v[0-9]+\.[0-9]+\.[0-9]+(-prerelease)?$/'
+      when: on_success
+  script:
+    - ../.gitlab/build-deb-rpm.sh
+
+.release-package:
+  stage: deploy
+  variables:
+    PRODUCT_NAME: auto_inject-dotnet
+    PACKAGE_FILTER: dotnet
```

Comment:
```suggestion
```

This was only necessary for nodejs because the package name was different from the product name